tests/functional/features/login.js

```diff
@@ -0,0 +1,63 @@
+import { genEmail, getLocation, pages, SITE } from '../helpers'
+import { HomePage } from '../page-objects/home-page'
+import { LoginRegisterPage } from '../page-objects/login-register-page'
+import { DashboardPage } from '../page-objects/dashboard-page'
+
+const { login, signUp, signUpNoDisclaimer } = LoginRegisterPage
+const password = 'Foo@1234foo'
+
+fixture `User login`
+  .beforeEach(async t => {
+    t.ctx.userEmail = genEmail('foo')
```

Comment:
Can each test use `getEmail` and pass it to the necessary functions instead of placing it on a context object?

Reply:
doh, I meant `genEmail`, not `getEmail`. Thought I edited my comment last night to fix, but I guess not.